### PR TITLE
Add support for the Kudu datastore in GoraExplorer

### PR DESCRIPTION
**Pull request by mgov88**
Add metadata analyzer feature in KUDU for GoraExplorer support.

### CODE REVIEW

File: gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduTableMetadata.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * limitations under the License.
+ */
+package org.apache.gora.kudu.store;
+
+import java.util.Map;
+
+public class KuduTableMetadata {
+
+  //Primary key
+  private String primaryKey;
```
Comment (alfonsonishikawa) [inline gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduTableMetadata.java:24]: From the example mapping in tests, the primary key column has a type. Can only be "STRING" or we can get the type from kudu?

File: gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduStoreMetadataAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  public List<String> getTablesNames() throws GoraException {
+    try {
+      return client.getTablesList().getTablesList();
+    } catch (KuduException ex) {
+      throw new GoraException(ex);
+    }
+  }
+
+  @Override
+  public Object getTableInfo(String tableName) throws GoraException {
```
Comment (alfonsonishikawa) [inline gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduStoreMetadataAnalyzer.java:78]: This should return KuduTableMetadata :)

**Review by alfonsonishikawa — CHANGES_REQUESTED** [review]
I added two coments :)

File: gora-kudu/pom.xml [hunk 3]
```diff
@@ -93,7 +93,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <skipTests>true</skipTests>
+          <skipTests>false</skipTests>
```
Comment (jhnmora000) [inline gora-kudu/pom.xml:96]: The tests of Kudu fail on Jenkins (https://issues.apache.org/jira/browse/GORA-636), please undo this change.

File: gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduTableMetadata.java (hunk 1 above)
Comment (mgov88) [reply gora-kudu/src/main/java/org/apache/gora/kudu/store/KuduTableMetadata.java:24]: it is only the name of the PK column, the column metadata is in the columns hashmap, should I put this in other variable?

File: gora-kudu/pom.xml (hunk 3 above)
Comment (mgov88) [reply gora-kudu/pom.xml:96]: ok thanks did know that

**Review by alfonsonishikawa — CHANGES_REQUESTED** [review]

**Comment by alfonsonishikawa** [comment]
Merged into master. Thank you, @mgov88 !
